docs/index.rst

```diff
@@ -1,8 +1,8 @@
-Raiden Network Documentation v\ |version|
+Raiden Network |capitalized_version| Documentation
 *****************************************
 
 
-This is the documentation for v\ |version| of the `Raiden Network <http://raiden.network/>`_. You can find the source code in `our Github Repository <https://github.com/raiden-network/raiden>`_.
+This is the documentation for |version| of the `Raiden Network <http://raiden.network/>`_. You can find the source code in `our Github Repository <https://github.com/raiden-network/raiden>`_.
```

Comment:
The first sentence sounds bit strange. In the docs this becomes
> This is the documentation for nightly of the Raiden Network

or
> This is the documentation for 0.2.0 of the Raiden Network

Maybe something like:
- This is the documentation for **the 0.2.0 version** of the Raiden Network
- This is the documentation for **version 0.2.0** of the Raiden Network

Reply:
done